com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md

```diff
@@ -7,7 +7,12 @@ This document describes the format of the mask map and detail map so that you ca
 To create a mask map:
 
 1. Open image editing software that supports channel editing (such as Adobe Photoshop).
-2. Drag your grayscale textures into their respective color channel. For information about which texture belongs in which channel, see [mask map](#MaskMap) and [detail map](#DetailMap).<br />![](Images/MaskMapAndDetailMap1.png)
+2. Import your grayscale textures onto separate layers.
+3. Make only one of the layers visble.
```

Comment:
This would by better phrased by telling the user to hide all layers except one. (I still don't understand why we have multiple layers in the first place though :D)